spec/docs/advanced.md

```diff
@@ -64,3 +64,76 @@ For now, it is unclear how to handle a nested function where that nested Triples
             rml:inputValue "-"
         ] .
 ```
+
+### Conditions
+
+Conditions are a shortcut to make RML documents more intuitive, but rely on existing FNML functionality.
+
+```turtle "example": "use condition"
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix grel: <http://users.ugent.be/~bjdmeest/function/grel.ttl#> .
+@prefix rml: <http://w3id.org/rml/> .
+
+<#Person_Mapping>
+    rml:logicalSource <#LogicalSource> ;
+    rml:subjectMap <#SubjectMap> ;
+    rml:predicateObjectMap <#NameMapping> .
+
+# Suggestion: add rml:condition predicate to expression map,
```

Comment:
avoid comments inside and move them outside the example?

Reply:
fixed